src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -9,7 +9,7 @@ metaDescription: How to install the K8 integration using Helm. Alternative metho
 
```

Comment:
the meta description (few lines above) is `How to install the K8 integration using Helm. Alternative method to New Relic's default install for infrastructure monitoring.` However, that is the default now

Reply:
The page in the dropdown is marked as an alternative
![Screen Shot 2021-08-25 at 4 06 54 PM](https://user-images.githubusercontent.com/43335750/130805775-195f08d2-847a-4d37-af69-ad3e92aa5e1e.png)

Reply:
or is it the alternative with respect to going through the wizard?

Reply:
I think "Manual" would be the best choice here, let me change that.